LeezeIt/LeezeIt/TextField.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  TextField.swift
+//  LeezeIt
+//
+//  Created by Riffat Khan on 9/6/22.
+//
+
+import Foundation
+import UIKit
+
+// adding max lenght attribute on interface builder
+private var __maxLengths = [UITextField: Int]()
+
+extension UITextField {
+    @IBInspectable var maxLength: Int {
+        get {
+            guard let l = __maxLengths[self] else {
+               return 150 // (global default-limit. or just, Int.max)
+            }
+            return l
+        }
+        set {
+            __maxLengths[self] = newValue
```

Comment:
why are you use underscores before maxLength?

Reply:
Resolved